doc/api/crypto.md

```diff
@@ -2143,15 +2143,23 @@ request.
 ### crypto.scrypt(password, salt, keylen[, options], callback)
 <!-- YAML
 added: v10.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/XXX
+    description: The `cost`, `blockSize` and `parallelization` option names
+                 have been added.
 -->
 - `password` {string|Buffer|TypedArray}
 - `salt` {string|Buffer|TypedArray}
 - `keylen` {number}
 - `options` {Object}
-  - `N` {number} CPU/memory cost parameter. Must be a power of two greater
-                 than one. **Default:** `16384`.
-  - `r` {number} Block size parameter. **Default:** `8`.
-  - `p` {number} Parallelization parameter. **Default:** `1`.
+  - `cost` {number} CPU/memory cost parameter. Must be a power of two greater
+                    than one. **Default:** `16384`.
```

Comment:
Such indentation produces false code blocks [in HTML version](https://nodejs.org/download/nightly/v11.0.0-nightly20180625c403eeb7fd/docs/api/crypto.html#crypto_crypto_scrypt_password_salt_keylen_options_callback).

`maxmem` case below will be fixed in https://github.com/nodejs/node/pull/21500, `cost` case seems to need fixing here.

Reply:
@vsemozhetbyt Thanks for pointing that out, done!